Observer/Order/Save.php

```diff
@@ -248,6 +249,7 @@ private function sendNewOrder(Order $order, AccountInterface $nostoAccount, Stor
             $nostoCustomerIdentifier = NostoOrderCreate::IDENTIFIER_BY_REF;
         }
         $nostoOrder = $this->nostoOrderBuilder->build($order);
+        $nostoOrder->setCustomer(new Buyer());
```

Comment:
Please add a line of comment why we are setting an empty customer here rather than removing it from the builder.